src/configureWorkspace/configurePython.ts

```diff
@@ -3,68 +3,224 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { getComposePorts, getExposeStatements, IPlatformGeneratorInfo, PackageInfo } from './configure';
+import * as path from "path";
+import vscode = require('vscode');
+import { TelemetryProperties } from 'vscode-azureextensionui';
+import { DockerDebugScaffoldContext } from '../debugging/DebugHelper';
+import { dockerDebugScaffoldingProvider, PythonScaffoldingOptions } from '../debugging/DockerDebugScaffoldingProvider';
+import { ext } from "../extensionVariables";
+import { PythonExtensionHelper } from '../tasks/python/PythonExtensionHelper';
+import { getPythonProjectType, inferPythonArgs, PythonDefaultDebugPort, PythonDefaultPorts, PythonFileTarget, PythonModuleTarget, PythonProjectType } from "../utils/pythonUtils";
+import { getComposePorts, getExposeStatements } from './configure';
+import { ConfigureTelemetryProperties, genCommonDockerIgnoreFile, quickPickGenerateComposeFiles } from './configUtils';
+import { ScaffolderContext, ScaffoldFile } from './scaffolding';
 
-export let configurePython: IPlatformGeneratorInfo = {
-    genDockerFile,
-    genDockerCompose,
-    genDockerComposeDebug,
-    defaultPorts: [3000],
-    initializeForDebugging: undefined,
-};
+interface LaunchFilePrompt {
+    prompt: string,
+    defaultFile: string
+}
 
-function genDockerFile(serviceNameAndRelativePath: string, platform: string, os: string | undefined, ports: number[], { cmd, author, version, artifactName }: Partial<PackageInfo>): string {
-    let exposeStatements = getExposeStatements(ports);
+const defaultLaunchFile: Map<PythonProjectType, LaunchFilePrompt> = new Map<PythonProjectType, LaunchFilePrompt>([
+    ["django", { prompt: "Enter the relative path to the application (e.g. manage.py)", defaultFile: "manage.py" }],
```

Comment:
Nit: make prompts more consistent (parens vs. no parens).  Also, we've been encouraged not to use "e.g." in user text, preferring "for example".

Reply:
The idea is to be similar to the prompts in the Python extension, so as to make the experience more familiar to the user. I'll chat with Uche on that since he suggested the format above.

Reply:
I added the parentheses for consistency, but will keep the wording till we get Uche's feedback.